fast64_internal/f3d/f3d_writer.py

```diff
@@ -2028,11 +2031,14 @@ def saveBitGeo(value, defaultValue, flagName, setGeo, clearGeo, matWriteMethod):
 	if value != defaultValue or matWriteMethod == GfxMatWriteMethod.WriteAll:
 		if value:
 			setGeo.flagList.append(flagName)
-		else:
+		elif matWriteMethod != GfxMatWriteMethod.WriteAll:
 			clearGeo.flagList.append(flagName)
 
 def saveGeoModeDefinition(fMaterial, settings, defaults, matWriteMethod):
-	setGeo = SPSetGeometryMode([])
+	if matWriteMethod == GfxMatWriteMethod.WriteAll:
+		setGeo = SPLoadGeometryMode([])
```

Comment:
`SPLoadGeometryMode` is only in f3dex2, so this should actually be made f3dex2-specific somehow

(and if using f3dex2 it would be better to use `gsSPGeometryMode(clear, set)` in both write methods, but that would be more involved with regards to fast64 code)